.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,7 +13,8 @@
 - [ ] Client: I added `authorities` to all new routes and check the course groups for displaying navigation elements (links, buttons)
 - [ ] Client: I documented the TypeScript code using JSDoc style.
 - [ ] Client: I added multiple screenshots/screencasts of my UI changes
-- [ ] Client: I translated all the newly inserted strings into English and German using neutral, gender-inclusive language
+- [ ] Client: I translated all newly inserted strings into English and German.
+- [ ] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/language-guidelines.html).
```

Comment:
AFAIK the Link to the documentation should not point to readthedocs, but docs.artemis.....

Reply:
hm... other guidelines (server, client) mentioned in this template use the same reference....
`https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html`
`https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/server.html`
But I am not sure, if/how I should address the `Found reference to undefined definition` warning that is indicated here.

Reply:
The documentation is deployed to https://docs.artemis.ase.in.tum.de/

Therefore, I think you have to use https://docs.artemis.ase.in.tum.de/dev/guidelines/client/ or something like that

Reply:
We recently deployed the docs to a new website, so it would be great if you can update the URL for the other places as well

Reply:
done with d25fed1, thanks for the hint @dfuchss